### PR TITLE
dev-python/snakemake-3.4.2: new ebuild

### PR DESCRIPTION
**Pull request by mschubert**
> Snakemake is a workflow management system that aims to reduce the complexity of creating workflows by providing a fast and comfortable execution environment, together with a clean and modern specification language in python style. Snakemake workflows are essentially Python scripts extended by declarative code to define rules. Rules describe how to create output files from input files.

Note: my `repo-commit` fails with the error below, and I'm not sure why.

```bash
>>> Creating Manifest for /nfs/research2/software/prefix/var/lib/layman/science/dev-python/snakemake
  dependency.bad [fatal]        50
   dev-python/snakemake/snakemake-3.4.2.ebuild: DEPEND: ~amd64(default/linux/amd64/13.0)
[     '>=dev-lang/python-3.3.2-r2:3.3',
      'dev-lang/python:3.4',
      'dev-lang/python:3.5']
   dev-python/snakemake/snakemake-3.4.2.ebuild: RDEPEND: ~amd64(default/linux/amd64/13.0)
...50 more...
```

### CODE REVIEW

File: dev-python/snakemake/snakemake-3.4.2.ebuild [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+DESCRIPTION="Make-like task language"
+HOMEPAGE="https://bitbucket.org/johanneskoester/snakemake/wiki/Home"
+SRC_URI="mirror://pypi/${PN:0:1}/${PN}/${P}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~x86 ~amd64-linux ~x86-linux"
+
+REQUIRED_USE="${PYTHON_REQUIRED_USE}"
```
Comment (jlec) [inline dev-python/snakemake/snakemake-3.4.2.ebuild]: unnecessary with distutils-r1.eclass.

Comment (jlec) [inline dev-python/snakemake/snakemake-3.4.2.ebuild]: > unnecessary with distutils-r1.eclass. 
Perhpas this also fixes your repo-commit thing.

**Comment by mschubert** [comment]
Same error after removing the `PYTHON_REQUIRED_USE` line.

I remember I had this before, solution was to specify `gentoo_prefix` instead of `gentoo` as master. But now it is back again.

**Comment by jlec** [comment]
It's fine for me as well as for travis. So I am merging.

**Pull request merged**